app/models/subscription.rb

```diff
@@ -0,0 +1,76 @@
+class Subscription < ApplicationRecord
+  include Auditor::Model
+
+  enum status: %i[active trashed]
```

Comment:
I don't think we need this anymore, we loop over all subscriptions and deleted them when they're expired

Reply:
I see, we can bosh the concept of `ongoing` and `status` because we ended up hard deleting records instead of the original plan to soft delete.